src/datumaro/experimental/legacy.py

```diff
@@ -514,6 +514,53 @@ def convert_annotations(
         return result
 
 
+class ForwardLabelAnnotationConverter(ForwardAnnotationConverter):
+    """Forward converter for Label (single label classification) annotations."""
+
+    def __init__(self, label_attribute: AttributeInfo):
+        """Initialize with label attribute."""
+        super().__init__()
+        self.label_attribute = label_attribute
+
+    @classmethod
+    def create(cls, dataset: LegacyDataset) -> "ForwardLabelAnnotationConverter | None":
+        """Create converter instance for label annotations."""
+        categories = dataset.categories()
+        legacy_label_categories = categories.get(AnnotationType.label, None)
+
+        if legacy_label_categories is None:
+            return None
+
+        new_label_categories = LabelCategories()
+        for label_item in legacy_label_categories.items:
+            new_label_categories.add(label_item.name)
+
+        label_attribute = AttributeInfo(
+            type=int,
+            annotation=label_field(),
+            categories=new_label_categories,
+        )
+
+        return cls(label_attribute=label_attribute)
+
+    @classmethod
+    def get_annotation_type(cls) -> AnnotationType:
+        return AnnotationType.label
+
+    def get_schema_attributes(self) -> dict[str, AttributeInfo]:
+        return {"label": self.label_attribute}
+
+    def convert_annotations(
+        self, annotations: list[Annotation], item: DatasetItem
+    ) -> dict[str, Any]:
+        labels = [ann for ann in annotations if isinstance(ann, Label)]
+        result = {}
+        if len(labels) > 0:
+            # For single label classification, take the first label
+            result["label"] = labels[0].label
```

Comment:
If there are no labels, set to None to avoid crashes in other parts of the code.
```suggestion
            result["label"] = labels[0].label
        else:
            result["label"] = None
```

Reply:
fixed